Module not found: Error: Can't resolve 'v8'

## Bug Report

<!-- If you would like to implement a PR, we are more than happy to help you go through the process!-->
- [ ] I would like to work on a fix!

```
* v8 in ./node_modules/@babel/core/lib/transformation/util/clone-deep.js

To install it, you can run: npm install --save v8
Child
    
    ERROR in ./node_modules/@babel/core/lib/transformation/util/clone-deep.js
    Module not found: Error: Can't resolve 'v8' in '/Users/qifutao/Documents/workspace/dxp-flow/frontend/node_modules/@babel/core/lib/transformation/util'
     @ ./node_modules/@babel/core/lib/transformation/util/clone-deep.js 16:9-22
     @ ./node_modules/@babel/core/lib/transformation/normalize-file.js
     @ ./node_modules/@babel/core/lib/transformation/index.js
     @ ./node_modules/@babel/core/lib/transform-ast.js
     @ ./node_modules/@babel/core/lib/index.js
     @ ./node_modules/@whalecloud/page-builder/utils/transformJS.js
     @ ./node_modules/@whalecloud/page-builder/render/components/RenderForm.js
     @ ./node_modules/@whalecloud/page-builder/render/components/FormRender.js
     @ ./src/pages/approveList/index.tsx
     @ ./src/.fishx/config/routes.js
     @ ./src/.fishx/router.tsx
     @ ./src/.fishx/index.tsx
     @ multi ./src/.fishx/index.tsx
```

**Environment**
<!--- Tip: Instead of filling out the questions below, you can run `npx envinfo --preset babel` and paste the result below ``` -->
```

```
- Babel version(s): v7.13.13
- Node/npm version: Node 12
- OS: [e.g. macOS 10.15.4]
- Monorepo: yes
- How you are using Babel: `webpack`

**Possible Solution**
<!--- If you have suggestions on a fix for the bug -->

**Additional context**
<!-- Add any other context about the problem here. Or a screenshot if applicable -->

--- a/packages/babel-core/package.json
+++ b/packages/babel-core/package.json
@@ -41,6 +41,7 @@
     "./lib/config/files/index.js": "./lib/config/files/index-browser.js",
     "./lib/config/resolve-targets.js": "./lib/config/resolve-targets-browser.js",
     "./lib/transform-file.js": "./lib/transform-file-browser.js",
+    "./lib/transformation/util/clone-deep.js": "./lib/transformation/util/clone-deep-browser.js",
     "./src/config/files/index.js": "./src/config/files/index-browser.js",
     "./src/config/resolve-targets.js": "./src/config/resolve-targets-browser.js",
     "./src/transform-file.js": "./src/transform-file-browser.js",

